### PR TITLE
dev/reference, v3.0/reference, v2.1/reference:  add the downstream permission for lightning

### PR DESCRIPTION
**Pull request by ericsyh**
### What is changed, added or deleted? 

TiDB Lightning will require the permission in downstream TiDB while only FAQ introduce it and deploy doc has no mention about it. 
So I add the downstream permission in the Lightning deploy doc. 
Also change the name from TiDB-Lightning to TiDB Lightning in documents.

### Which version does your change affect? 

Apply in dev, v3.0, v2.1

### Checklist 

- [ ] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/reference/tools/tidb-lightning/deployment.md [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
 
-- TiDB-Lightning 运行后，TiDB 集群将无法正常对外提供服务。
+- TiDB Lightning 运行后，TiDB 集群将无法正常对外提供服务。
 - 若 `tidb-lightning` 崩溃，集群会留在“导入模式”。若忘记转回“普通模式”，集群会产生大量未压缩的文件，继而消耗 CPU 并导致迟延 (stall)。此时，需要使用 `tidb-lightning-ctl` 手动将集群转回“普通模式”：
 
     ```sh
     bin/tidb-lightning-ctl -switch-mode=normal
     ```
 
+- TiDB Lightning 需要下游 TiDB 的权限
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/deployment.md]:
```suggestion
- TiDB Lightning 需要下游 TiDB 的权限：
```

File: dev/reference/tools/tidb-lightning/deployment.md [hunk 2]
```diff
@@ -1,23 +1,35 @@
...
+
+  | 权限 | 作用域 |
+  |----:|:------|
+  | SELECT | Tables |
+  | INSERT | Tables |
+  | UPDATE | Tables |
+  | DELETE | Tables |
+  | CREATE | Databases,tables |
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/deployment.md]:
```suggestion
  | CREATE | Databases, tables |
```

File: v2.1/reference/tools/tidb-lightning/deployment.md [hunk 3]
```diff
@@ -1,23 +1,35 @@
...
 
-- TiDB-Lightning 运行后，TiDB 集群将无法正常对外提供服务。
+- TiDB Lightning 运行后，TiDB 集群将无法正常对外提供服务。
 - 若 `tidb-lightning` 崩溃，集群会留在“导入模式”。若忘记转回“普通模式”，集群会产生大量未压缩的文件，继而消耗 CPU 并导致迟延 (stall)。此时，需要使用 `tidb-lightning-ctl` 手动将集群转回“普通模式”：
 
     ```sh
     bin/tidb-lightning-ctl -switch-mode=normal
     ```
 
+- TiDB Lightning 需要下游 TiDB 的权限
```
Comment (yikeke) [inline v2.1/reference/tools/tidb-lightning/deployment.md]:
```suggestion
- TiDB Lightning 需要下游 TiDB 的权限：
```

File: v2.1/reference/tools/tidb-lightning/deployment.md [hunk 4]
```diff
@@ -1,23 +1,35 @@
...
+
+  | 权限 | 作用域 |
+  |----:|:------|
+  | SELECT | Tables |
+  | INSERT | Tables |
+  | UPDATE | Tables |
+  | DELETE | Tables |
+  | CREATE | Databases,tables |
```
Comment (yikeke) [inline v2.1/reference/tools/tidb-lightning/deployment.md]:
```suggestion
  | CREATE | Databases, tables |
```

File: v3.0/reference/tools/tidb-lightning/deployment.md [hunk 5]
```diff
@@ -1,24 +1,36 @@
...
 
-- TiDB-Lightning 运行后，TiDB 集群将无法正常对外提供服务。
+- TiDB Lightning 运行后，TiDB 集群将无法正常对外提供服务。
 - 若 `tidb-lightning` 崩溃，集群会留在“导入模式”。若忘记转回“普通模式”，集群会产生大量未压缩的文件，继而消耗 CPU 并导致迟延 (stall)。此时，需要使用 `tidb-lightning-ctl` 手动将集群转回“普通模式”：
 
     ```sh
     bin/tidb-lightning-ctl -switch-mode=normal
     ```
 
+- TiDB Lightning 需要下游 TiDB 的权限
```
Comment (yikeke) [inline v3.0/reference/tools/tidb-lightning/deployment.md]:
```suggestion
- TiDB Lightning 需要下游 TiDB 的权限：
```

File: v3.0/reference/tools/tidb-lightning/deployment.md [hunk 6]
```diff
@@ -1,24 +1,36 @@
...
+
+  | 权限 | 作用域 |
+  |----:|:------|
+  | SELECT | Tables |
+  | INSERT | Tables |
+  | UPDATE | Tables |
+  | DELETE | Tables |
+  | CREATE | Databases,tables |
```
Comment (yikeke) [inline v3.0/reference/tools/tidb-lightning/deployment.md]:
```suggestion
  | CREATE | Databases, tables |
```

**Comment by yikeke** [comment]
Please also ask someone to take a technical review, thanks~ @ericsyh

**Comment by ericsyh** [comment]
@kennytm Kenny PTAL~

File: dev/reference/tools/tidb-lightning/deployment.md [hunk 7]
```diff
@@ -1,23 +1,35 @@
...
+
+  | 权限 | 作用域 |
+  |----:|:------|
+  | SELECT | Tables |
+  | INSERT | Tables |
+  | UPDATE | Tables |
+  | DELETE | Tables |
+  | CREATE | Databases, tables |
+  | DROP | Databases, tables |
+  | ALTER | Tables |
```
Comment (kennytm) [inline dev/reference/tools/tidb-lightning/deployment.md]: Executing ADMIN CHECKSUM TABLE requires the user to be administrator 🤔

Comment (yikeke) [reply dev/reference/tools/tidb-lightning/deployment.md]: Please resolve the comment. @ericsyh

**Pull request closed without merging**